power_grid_model_c/power_grid_model/include/power_grid_model/optimizer/tap_position_optimizer.hpp

```diff
@@ -598,17 +598,130 @@ class TapPositionOptimizerImpl<std::tuple<TransformerTypes...>, StateCalculator,
     static constexpr auto transformer_index_of = container_impl::get_cls_pos_v<T, TransformerTypes...>;
     static_assert(((transformer_index_of<TransformerTypes> < sizeof...(TransformerTypes)) && ...));
 
+    struct BinarySearch {
```

Comment:
the structure, formatting, typesetting, ... of this class is generally quite different from the existing conventions in the PGM. is there a particular reason for that? is it a proposal or an accidental WIP thing?

Reply:
comment addressed in the latest commit.